multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -25,7 +25,7 @@ externalDNSName: {{.ExternalDNSName}}
 #hostedZone: ""
```

Comment:
Add deprecation note to comment comment above (people shouldn't need to wait until running to find out it's deprecated)